.yarnrc.yml

```diff
@@ -1,13 +1,9 @@
+compressionLevel: mixed
+
 enableGlobalCache: true
 
 httpTimeout: 300000
 
 nodeLinker: node-modules
 
 npmRegistryServer: 'https://registry.npmjs.org/'
-
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
-    spec: '@yarnpkg/plugin-workspace-tools'
-
-yarnPath: .yarn/releases/yarn-3.8.1.cjs
```

Comment:
FYI, You can still use path with yarn 4. In my case using only corepack caused some issues on CI.

Reply:
looking on the failing CI, some install gets changes...
have in minnd you need to run a full yarn install, without `-mode=update-lockfile`

Reply:
just dropping some  issues i encounter on the community.plugins to have in mind

Reply:
Thanks for the input. 

> FYI, You can still use path with yarn 4. In my case using only corepack caused some issues on CI.

This should be addressed with my PR to the `yarn-install` action from `backstage/actions`

> looking on the failing CI, some install gets changes...
have in minnd you need to run a full yarn install, without -mode=update-lockfile

Yes this is why the lockfile is regenerated mostly. 

> yarn --cwd  stopped working on yarn 4 too.. so instead of yarn --cwd  ./path command now need to run yarn ./path command

Yes, addressed with using [`working-directory` ](https://github.com/backstage/backstage/pull/28881/files#diff-e0f03fe606d54c0a66c7afc5ba8c9452e10023b66772486085510fdbd0403513R36)

Reply:
I'd prefer to keep using `yarnPath`, less headaches that way

Reply:
You mean in this PR or generally? 

In my experience it having the code checked in rather creates headache in the upgrade process. 

[Yarn itself seems to agree. ](https://yarnpkg.com/blog/release/4.0#installing-yarn)

> To that end we used to recommend using the [yarnPath](https://yarnpkg.com/configuration/yarnrc#yarnPath) setting pointing to a checked-in binary, but this pattern increased friction more than we liked - many people didn't like the idea of adding a binary to their repository, however small. We listened, and worked conjointely with Node.js on a project called [Corepack](https://nodejs.org/api/corepack.html). Corepack is a tool shipped with Node.js 16+ that will automatically select the right package manager version to run depending on the project you're working on.